File: Challenge/FIBONACCI/fibonacci.py
def fibonacci(n):
    fib_sequence = [0, 1]

    while len(fib_sequence) < n:
        fib_sequence.append(fib_sequence[-1] + fib_sequence[-2])

    return fib_sequence[:n]

# Imprimir los primeros 50 números de Fibonacci
primeros_50_fibonacci = fibonacci(50)
print(primeros_50_fibonacci)

File: Challenge/FIBONACCI/fibonacci.kt
fun fibonacci(n: Int): List<Long> {
    val fibSequence = mutableListOf(0L, 1L)

    while (fibSequence.size < n) {
        val nextNumber = fibSequence[fibSequence.size - 1] + fibSequence[fibSequence.size - 2]
        fibSequence.add(nextNumber)
    }

    return fibSequence.take(n)
}

fun main() {
    // Imprimir los primeros 50 números de Fibonacci
    val primeros50Fibonacci = fibonacci(50)
    println(primeros50Fibonacci)
}
